samples/wifi/scan/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _wifi_scan_sample:
+
+Wi-Fi: Scan
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample allows the Nordic Wi-Fi chipset to scan for the Access points.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+This sample can perform Wi-Fi operation "Scan".
+
+Using this sample, the development kit can scan for access points in :abbr:`STA (Station)` mode.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/scan`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Currently, the following configurations are supported:
+
+* 7002 DK + QSPI
```

Comment:
Replace this with -
Currently, the following board(s) are supported :
* nRF7002 DK